concent_api/core/payments/backends/sci_backend.py

```diff
@@ -145,6 +153,21 @@ def call_on_confirmed_transaction(
     )
 
 
+def get_covered_additional_verification_costs(client_eth_address: str, payment_ts: int) -> list:
+    assert isinstance(client_eth_address, str) and len(client_eth_address) == ETHEREUM_ADDRESS_LENGTH
```

Comment:
I saw in couple of Your pull requests such checks. Consider create function e.g. `validate_ethereum_address` and use this instead of this checks. There are different approaches to asserts, for me some of them are OK, but you could extract low level logic to separate, even one line function and use it instead. I don't say that is must be done now, please consider and decide.

Advantage of this is as well, when something will change (e.g. `payment_ts >= 0 -> payment_ts >= ??` it will be necessary to change it in only one place

Reply:
`validate_ethereum_address()` suggests that it would be a function that checks the condition and raises an exception. For assertions we need simple boolean conditions instead.

Also these are two separate checks. I would do them with separate assertions:
``` python
assert isinstance(client_eth_address, str)
assert len(client_eth_address) == ETHEREUM_ADDRESS_LENGTH
```

And these checks are so simple that wrapping them in functions could only make things less clear.

Reply:
> Advantage of this is as well, when something will change (e.g. payment_ts >= 0 -> payment_ts >= ?? it will be necessary to change it in only one place

This can be also partially achieved by using constants.